npm/README.md

```diff
@@ -0,0 +1,15 @@
+# NPM rule usage guide
```

Comment:
Here we explain how to generate an auth token, because it's now a requirement to use `deploy_npm`.

I don't expect these instructions to change in the next couple of years.